eva/optimizer/plan_generator.py

```diff
@@ -84,6 +84,11 @@ def optimize(self, logical_plan: Operator):
 
         # Build Optimal Tree
         optimal_plan = self.build_optimal_physical_plan(root_grp_id, optimizer_context)
+
+        #map columns to ids
```

Comment:
Please elaborate the comment

Reply:
updated it. please let me know if its okay